app/views/candidate_interface/gcse/english/grade/edit.html.erb

```diff
@@ -19,20 +19,16 @@
       <% else %>
         <%= render CandidateInterface::GcseGradeGuidanceComponent.new(@subject, @qualification_type) %>
 
-        <% if autocomplete_grades? %>
-          <div class="govuk-!-width-one-third">
-            <%= f.govuk_collection_select(
-                  :grade,
-                  CandidateInterface::GcseQualificationDetailsForm.all_grade_drop_down_options,
-                  :value,
-                  :option,
-                  label: { text: 'Grade', size: 'm' },
-                  hint: { text: 'For example, ‘C’ or ‘4’' },
-                  ) %>
-          </div>
-        <% else %>
-          <%= f.govuk_text_field :grade, label: { text: t('application_form.gcse.grade.label'), size: 'm' }, hint: { text: hint_for_gcse_edit_grade(@subject, @qualification_type) }, width: 10 %>
-        <% end %>
+        <div class="govuk-!-width-one-third">
```

Comment:
This view will eventually be deleted once we remove the relevant multiple GCSE feature flag